example-apps/relevance-workbench/README.md

```diff
@@ -10,7 +10,7 @@ You can also try with your own data by forking this repo and plugging the applic
 
 ## Pre-requisites
 
-To run this demo successfully, you will need an Elasticsearch deployment (> 8.8) with the ELSER model deployed. The easiest way for this is to use Elastic Cloud as described in the next part but you can also deploy Elasticsearch locally.
+To run this demo successfully, you will need an Elasticsearch deployment (> 8.11) with the ELSER model deployed. The easiest way for this is to use Elastic Cloud as described in the next part but you can also deploy Elasticsearch locally.
```

Comment:
Should we also point out that >8.8 has ELSERv1, and >8.11 has ELSERv2? We might want users that have not been able to upgrade yet to be able to use this notebook

Reply:
Maybe we should just have _separate_ notebooks for simplicity? v1 and v2, with a big info banner at top of each explaining version availability etc.?

Reply:
I like that - we could have a "if you're using version < 8.11, then go here"

Reply:
Based on the outcome of discussion in [thread](https://elastic.slack.com/archives/C05E3NSDLS2/p1697556832684659?thread_ts=1696523296.981769&cid=C05E3NSDLS2), we are not adding/updating notebook(s) to support ELSER v1 model.